src/components/atoms/BackgroundContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Flex } from 'rebass';
+
+interface BackgroundContainerPropsType {
```

Comment:
이미 타입인게 명확해서 포스트픽스로 Type을 안 붙여줘도 되지 않을까요? (중복되는 느낌)
```suggestion
interface BackgroundContainerProps {
```

Reply:
오호 이 부분 다른 컴포넌트 타입명도 함께 수정해서 커밋하겠읍니다!

Reply:
`ff026df`에서 반영완료했습니다!